File: Csharp/Behavioral/Command/Example/Command/Receiver.cs
using System;

namespace Command
{
    public class Receiver
    {
        public void DoSomething(string a)
        {
            Console.WriteLine($"Receiver: Relieved first item ({a}.)");
        }

        public void DoSomethingElse(string b)
        {
            Console.WriteLine($"Receiver: Received Second Item ({b}.)");
        }
    }
}

File: PHP/Behavioral/Command/Example1/Receiver.php
<?php
class Receiver
{
    public function doSomething(string $a): void
    {
        echo "Receiver: Working on (" . $a . ".)<br/>";
    }

    public function doSomethingElse(string $b): void
    {
        echo "Receiver: Also working on (" . $b . ".)<br/>";
    }
}
